diff --git a/etc/bashrc b/etc/bashrc
index 18f1cdae8..a9deab387 100644
--- a/etc/bashrc
+++ b/etc/bashrc
@@ -212,12 +212,12 @@ _foamSource $WM_PROJECT_DIR/etc/config.sh/aliases
 # Source user setup files for optional packages
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 _foamSource `$WM_PROJECT_DIR/bin/foamEtcFile config.sh/mpi`
-if _foam_on_macos
-then
-    _foamSource `$WM_PROJECT_DIR/bin/foamEtcFile config.sh/mac/paraview`
-else
-    _foamSource `$WM_PROJECT_DIR/bin/foamEtcFile config.sh/paraview`
-fi
+# if _foam_on_macos
+# then
+#     _foamSource `$WM_PROJECT_DIR/bin/foamEtcFile config.sh/mac/paraview`
+# else
+#     _foamSource `$WM_PROJECT_DIR/bin/foamEtcFile config.sh/paraview`
+# fi
 _foamSource `$WM_PROJECT_DIR/bin/foamEtcFile config.sh/ensight`
 _foamSource `$WM_PROJECT_DIR/bin/foamEtcFile config.sh/gperftools`
 
diff --git a/etc/config.sh/mpi b/etc/config.sh/mpi
index 56bca560b..9f101966f 100644
--- a/etc/config.sh/mpi
+++ b/etc/config.sh/mpi
@@ -35,7 +35,7 @@ unset MPI_ARCH_PATH MPI_HOME FOAM_MPI_LIBBIN
 case "$WM_MPLIB" in
 SYSTEMOPENMPI)
     # Use the system installed openmpi, get library directory via mpicc
-    export FOAM_MPI=openmpi-system
+    # export FOAM_MPI=openmpi-system
 
     # Undefine OPAL_PREFIX if set to one of the paths on foamOldDirs
     if [ -z "$($foamClean "$OPAL_PREFIX" "$foamOldDirs")" ]
